receiver/flinkmetricsreceiver/testdata/integration/setup.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -e
+
+./usr/local/flink/bin/flink run --detached /integration/StateMachineExample.jar
```

Comment:
What is this jar? I don't think we're likely to commit this to the repo.

Reply:
found out it exists in the dockerfile too. can be removed. updated